arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -95,7 +113,10 @@ public boolean isAbortingScenario() {
   // last row
   @Setter private ContextFragment finalContextFragment;
 
+  private Address callerAddress;
```

Comment:
final

Reply:
We can make it final, but currently we only initialize this value once the CALL is unexceptional.